<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SectionTransformPanel</class>
 <widget class="QWidget" name="SectionTransformPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>339</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="transformationGroupBox">
     <property name="title">
      <string>Trasformazione</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_21">
      <item row="1" column="0" colspan="2">
       <widget class="QGroupBox" name="translateGroupBox">
        <property name="title">
         <string>Traslazione</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_20">
         <item row="0" column="1">
          <widget class="QLabelUnitMeasure" name="translateDyUMLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m]
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="2">
          <widget class="QPushButton" name="translatePButton">
           <property name="text">
            <string>Trasla</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEditValue" name="translateDyLEdit"/>
         </item>
         <item row="1" column="2">
          <widget class="QLineEditValue" name="translateDzLEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabelRichName" name="translateDzLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
Δz
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabelRichName" name="translateDyLabel">
           <property name="text">
            <string>&lt;html&gt;&lt;/head&gt;
&lt;body&gt;Δy&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabelUnitMeasure" name="translateDzUMLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[m]
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Rotazione</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabelRichName" name="rotateAngleLabel">
           <property name="text">
            <string>&lt;html&gt;&lt;/head&gt;
&lt;body&gt;α&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabelUnitMeasure" name="rotateAngleUMLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;/head&gt;
&lt;body&gt;
[rad]
&lt;/body&gt;
&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QLineEditValue" name="rotateAngleLEdit"/>
         </item>
         <item row="0" column="3">
          <widget class="QPushButton" name="rotateAnglePButton">
           <property name="text">
            <string>Ruota</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>57</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="PTransRotPButton">
     <property name="text">
      <string>Sistema centrale</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="GTranslatePButton">
     <property name="text">
      <string>Sistema baricetrico</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>85</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QLabelRichName</class>
   <extends>QLabel</extends>
   <header>qlabelrichname.h</header>
  </customwidget>
  <customwidget>
   <class>QLineEditValue</class>
   <extends>QLineEdit</extends>
   <header>qlineeditvalue.h</header>
  </customwidget>
  <customwidget>
   <class>QLabelUnitMeasure</class>
   <extends>QLabel</extends>
   <header>qlabelunitmeasure.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
